incubator/hnc/docs/releasing.md

```diff
@@ -91,7 +91,7 @@ Get the release ID. Sadly, this isn't available through the UI, so get it by
 calling:
 
 ```bash
-curl -u "$HNC_USER:$HNC_PAT" $MT_ENDPOINT/releases | less`
```

Comment:
What does this mean?

Reply:
That extra backtick was just a formatting error. If you're asking about the command as a whole, it's just a way to query the Github API to get the release IDs. I can clarify in another PR if you like.

Reply:
Thank you for the explanation!